execution/src/posix/execution.cc

```diff
@@ -279,7 +282,7 @@ namespace leatherman { namespace execution {
 
         // Set the process group; this will be used by the parent if we need to kill the process and its children
         if (setpgid(0, 0) == -1) {
-            char const* message = "failed to setpgid.";
+            char const* message = _("failed to setpgid.").c_str();
```

Comment:
I'm not sure this is safe; alse we could use `size` instead of running `strlen`. I'd rather we not try to hold onto a pointer from `c_str`.